html/components/_tooltip.scss

```diff
@@ -0,0 +1,134 @@
+////
+/// @group tooltip
+////
+
+.sprk-c-Tooltip__container {
+  position: relative;
+  display: inline-flex;
```

Comment:
Does the inline-flex on tooltip work in IE11 okay without the vendor prefixes? Think we need those here.

Reply:
I tested this again in IE 11 and we're good